.gitmodules

```diff
@@ -9,14 +9,13 @@
 	path = scipy/_lib/unuran
 	url = https://github.com/scipy/unuran.git
 	shallow = true
-[submodule "HiGHS"]
-	path = scipy/_lib/highs
-	url = https://github.com/scipy/highs
-	shallow = true
 [submodule "scipy/_lib/boost_math"]
 	path = scipy/_lib/boost_math
 	url = https://github.com/boostorg/math.git
 	shallow = true
 [submodule "scipy/_lib/array_api_compat"]
 	path = scipy/_lib/array_api_compat
 	url = https://github.com/data-apis/array-api-compat.git
+[submodule "subprojects/highs"]
```

Comment:
Given that this is a break from our previous convention, could you please add a comment here to the rationale in the meson docs
```suggestion
# $REASON, see
# https://mesonbuild.com/Subprojects.html#why-must-all-subprojects-be-inside-a-single-directory
[submodule "subprojects/highs"]
```